### PR TITLE
[OLD PR] Update README.md files to reflect 3 projects

### PR DESCRIPTION
**Pull request by levithomason**
## This PR has been moved over to here: https://github.com/microsoft/fluentui/pull/14959
Pending feedback has also been addressed. 

#### Pull request checklist

- [ ] Include a change request file using `$ yarn change`
- [ ] Add high-level ROADMAP link if we have it, or remove this heading
- [ ] Add missing "Used By" in table for web components.
- [ ] Add missing "Quick Start" links.
- [ ] Path to "react" package is confusing since it is called office-ui-fabric-react
- [ ] Path to "react-northstar" is confusing since it is called fluentui/react-northstar
- [ ] Path to React Northstar README.md is confusing since it goes to packages/fluentui, and not the actual package.
- [ ] Make package README.md consistent in headings, links, format, and tone.
- [ ] Add images for used by, uber list for the main README and smaller lists for each individual README.
  - [ ] Web components: ...
  - [ ] Northstar: Teams, ...
  - [ ] React: Office, ...

#### Description of changes

The README.md here doesn't reflect the nature of this repo, 3 distinct projects working toward a larger goal.  This PR updates the main README.md to reflect that format and leaves the individual project README.md files to explain their own project.

Here's the updated [**README.md**](https://github.com/microsoft/fluentui/blob/e096e0b453821570e49e042530a80e2c1ff33870/README.md).

#### Focus areas to test

Read through the top-level README.md and each project's README.md and ensure it reads well and feels cohesive.

### CODE REVIEW

**Comment by levithomason** [comment]
@chrisdholt, @dzearing could I get some 👀 on this? Thanks.

File: README.md [hunk 1]
```diff
@@ -1,140 +1,28 @@
...
 
-**The React-based front-end framework for building web experiences.**
+> The goal of these projects is to dedupe functionality and increase interoperability over time. Why are there two React versions? See the [FAQ on Fabric and Stardust](https://github.com/microsoft/fluentui/wiki/FAQ---Fabric-and-Stardust-to-Fluent-UI)
 
-[![npm version](https://badge.fury.io/js/%40fluentui%2Freact.svg)](https://badge.fury.io/js/%40fluentui%2Freact) [![Build Status](https://dev.azure.com/uifabric/fabricpublic/_apis/build/status/office-ui-fabric-react%20-%20PR?branchName=master)](https://dev.azure.com/uifabric/fabricpublic/_build/latest?definitionId=84&branchName=master)
+## Roadmap
 
-Fluent UI React is a collection of robust React-based components designed to make it simple for you to create consistent web experiences using the Fluent Design Language.
+Each project within the Fluent UI ecosystem has its own roadmap. Here, you can find a high-level roadmap for the entire effort itself.
```
Comment (chrisdholt) [inline README.md:23]: Is this missing a link? Perhaps it's a TODO...

Comment (chrisdholt) [reply README.md:23]: Ah, I see it inserted this *after* deleting all this code. Resolved!

File: README.md [hunk 2]
```diff
@@ -1,140 +1,28 @@
-:tada: The **Office UI Fabric React** project has evolved to **Fluent UI**. :tada:
+# Fluent UI Web
```
Comment (dzearing) [inline README.md:2]: We need to have a one or two sentence explanation of what the repo's purpose is here.

> Fluent UI web represents a collection of utilities, React components, and web components for building web applications.

File: README.md [hunk 3]
```diff
@@ -1,140 +1,28 @@
...
+| **Used By**     | Office                                                                                                                                                                                                                                            | Teams                                                                                                                                                                                                                                                                           | TODO                                                                                                                                                                                                                                                 |
```
Comment (chrisdholt) [inline README.md:10]: FYI, aware of this and will advise 👍

File: README.md [hunk 4]
```diff
@@ -1,140 +1,28 @@
...
-To build individual packages within the `packages/*` or `apps/*` folders, `cd` to the relevant folder and run `yarn build`. Note that because the packages are symlinked together, you must manage building dependencies in the right order, or use the `yarn buildto` script to build to the specific package you want.
-
-#### Advanced build commands
-
-There are more advanced build commands listed in the [Build Commands](https://github.com/microsoft/fluentui/wiki/Build-Commands) wiki page.
-
-### Testing
-
-For info about testing, see our [testing documentation](https://github.com/microsoft/fluentui/wiki/Testing).
+TODO: LINK to high-level roadmap
```
Comment (dzearing) [inline README.md:25]: remove or replcae

File: packages/office-ui-fabric-react/README.md [hunk 5]
```diff
@@ -1,3 +1,158 @@
...
+
+All files on the Fluent UI React GitHub repository are subject to the MIT license. Please read the License file at the root of the project.
+
+Usage of the fonts and icons referenced in Fluent UI React is subject to the terms of the [assets license agreement](https://aka.ms/fluentui-assets-license).
+
+## Changelog
+
+We use [GitHub Releases](https://github.com/blog/1547-release-your-software) to manage our releases, including the changelog between every release. View a complete list of additions, fixes, and changes on the [releases](https://github.com/microsoft/fluentui/releases) page.
+
+# Looking for Office UI Fabric React?
```
Comment (dzearing) [inline packages/office-ui-fabric-react/README.md:144]: Don't think this goes here.

**Comment by JustSlone** [comment]
@levithomason since you'll be out for a bit I'll picking up this PR and see it over the finish line

**Comment by JustSlone** [comment]
Moved this over to my fork PR here: https://github.com/microsoft/fluentui/pull/14959

**Pull request closed without merging**